composer.json

```diff
@@ -42,8 +42,42 @@
 	"autoload": {
 		"psr-4": {
 			"Croogo\\": "src",
+			"Croogo\\Acl\\": "Acl/src",
+			"Croogo\\Acl\\Test\\": "Acl/tests",
+			"Croogo\\Blocks\\": "Blocks/src",
+			"Croogo\\Blocks\\Test\\": "Blocks/tests",
+			"Croogo\\Comments\\": "Comments/src",
+			"Croogo\\Comments\\Test\\": "Comments/tests",
+			"Croogo\\Contacts\\": "Contacts/src",
+			"Croogo\\Contacts\\Test\\": "Contacts/tests",
 			"Croogo\\Core\\": "Core/src",
 			"Croogo\\Core\\Test\\": "Core/tests",
+			"Croogo\\Dashboards\\": "Dashboards/src",
+			"Croogo\\Dashboards\\Test\\": "Dashboards/tests",
+			"Croogo\\Example\\": "Example/src",
+			"Croogo\\Example\\Test\\": "Example/tests",
+			"Croogo\\Extensions\\": "Extensions/src",
+			"Croogo\\Extensions\\Test\\": "Extensions/tests",
+			"Croogo\\FileManager\\": "FileManager/src",
+			"Croogo\\FileManager\\Test\\": "FileManager/tests",
+			"Croogo\\Install\\": "Install/src",
+			"Croogo\\Install\\Test\\": "Install/tests",
+			"Croogo\\Menus\\": "Menus/src",
+			"Croogo\\Menus\\Test\\": "Menus/tests",
+			"Croogo\\Meta\\": "Meta/src",
+			"Croogo\\Meta\\Test\\": "Meta/tests",
+			"Croogo\\Nodes\\": "Nodes/src",
+			"Croogo\\Nodes\\Test\\": "Nodes/tests",
+			"Croogo\\Settings\\": "Settings/src",
+			"Croogo\\Settings\\Test\\": "Settings/tests",
+			"Croogo\\Taxonomy\\": "Taxonomy/src",
+			"Croogo\\Taxonomy\\Test\\": "Taxonomy/tests",
+			"Croogo\\Translate\\": "Translate/src",
+			"Croogo\\Translate\\Test\\": "Translate/tests",
+			"Croogo\\Users\\": "Users/src",
+			"Croogo\\Translate\\Test\\": "Users/tests",
```

Comment:
Yup. Although, those should only be included for testing. See autoload-dev: https://getcomposer.org/doc/04-schema.md#autoload-dev